contracts/staking/IStaking.sol

```diff
@@ -117,17 +117,19 @@ interface IStaking {
     //
```

Comment:
here ceiling is number of apps, sure we can decrease type but it's not tokens

Reply:
Ah, it's the number of apps, right. We can leave it as `uint256` or... limit to `uint8`? 2^8-1 is still too much but at least we give a hint that this number needs to be chosen carefully and not too large because of gas limitations.

I am fine either way.